lib/pivotal-tracker/account_membership.rb

```diff
@@ -0,0 +1,27 @@
+class PivotalTracker::AccountMembership
+  attr_accessor :kind, :id, :person, :account_id, 
```

Comment:
@wlffann I think these should be alphabetized, to follow the existing templates.